otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -24,6 +24,11 @@ data:
     networkobservabilityHubble = true
     networkobservabilityCilium = true
     prometheuscollectorhealth = false
+    controlplane-apiserver = true
```

Comment:
- Add a link to control plan metrics doc
- Update our documentation for minimal ingestion profile

Reply:
update ccp configs to add 1024 limit

Reply:
doc update for minimal ingestion profile -> https://github.com/MicrosoftDocs/azure-docs-pr/pull/264369/files